js/hollaex.js

```diff
@@ -254,6 +254,8 @@ module.exports = class hollaex extends Exchange {
                 'info': currency,
                 'name': name,
                 'active': active,
+                'deposit': undefined,
```

Comment:
We can use the info from `publicGetConstants` to fill in these values.

Please, always add the exchange response sample immediately after a network request if it's missing (immediately after the `await` line). 

The goal of this task is not to just add the fields, but also to make sure that they have proper values used from the exchange response, if available.

You can easily verify that with the `cli`:

```
node examples/js/cli hollaex currencies --verbose --debug
```

From the response sample below it becomes obvious that we can determine the values of `deposit` and `withdraw` in this case, because the exchange provides that information – they should not be `undefined` here.

Make sure to follow this formatting:

```JavaScript
{
    "coins":{
        "bch":{
            "id":4,
            "fullname":"Bitcoin Cash",
            "symbol":"bch",
            "active":true,
            "verified":true,
            "allow_deposit":true,
            "allow_withdrawal":true,
            "withdrawal_fee":0.0001,
            "min":0.001,
            "max":100000,
            "increment_unit":0.001,
            "logo":"https://bitholla.s3.ap-northeast-2.amazonaws.com/icon/BCH-hollaex-asset-01.svg",
            "code":"bch",
            "is_public":true,
            "meta":{},
            "estimated_price":null,
            "description":null,
            "type":"blockchain",
            "network":null,
            "standard":null,
            "issuer":"HollaEx",
            "withdrawal_fees":null,
            "created_at":"2019-08-09T10:45:43.367Z",
            "updated_at":"2021-12-13T03:08:32.372Z",
            "created_by":1,
            "owner_id":1
        },
    },
    "network":"https://api.hollaex.network"
}
```